tests/lib/lodSelection.cpp

```diff
@@ -20,21 +20,32 @@
 #define BOOST_TEST_MODULE LodSelection
 #include <boost/test/unit_test.hpp>
 
+#include <livre/core/dash/DashTree.h>
+#include <livre/core/data/DataSource.h>
+
+#include <livre/lib/data/MemoryDataSource.h>
+#include <livre/lib/render/SelectVisibles.h>
+#include <livre/lib/visitor/DFSTraversal.h>
+
 #include <livre/core/render/Frustum.h>
-#include <livre/core/maths/maths.h>
 
-uint32_t getLOD( const uint32_t minLOD,
-                 const uint32_t maxLOD,
-                 const float screenSpaceError,
-                 const livre::Vector3f& worldPos,
-                 const float cameraDist )
-{
+#include <lunchbox/pluginRegisterer.h>
+
+// Explicit registration required because the folder of the data source plugin is not
+// in the LD_LIBRARY_PATH of the test executable.
+lunchbox::PluginRegisterer< livre::MemoryDataSource > registerer;
```

Comment:
I remember the RH and Ubuntu was behaving differently in link time about registering the data sources ( I had a problem with uvf before thats why I have added ).